po/de.po

```diff
@@ -239,7 +251,7 @@ msgstr "Dateien empfangen"
 
 #: data/ui/preferences-device-panel.ui:490
 msgid "Save files to"
-msgstr ""
+msgstr "Dateien speichern nach"
```

Comment:
```suggestion
msgstr "Dateien speichern unter"
```
AFAIK, this wording is more common in German software translations.

Reply:
Yep, you are right.